deploy/kubernetes/scripts/install-minikube.sh

```diff
@@ -1,17 +1,23 @@
 #!/bin/bash
 
-curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.10.0/bin/linux/amd64/kubectl \
+MINIKUBE_VERSION=v0.35.0
+KUBE_VERSION=v1.13.4
+
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl \
   && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
 
-curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-linux-amd64 \
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64 \
   && chmod +x minikube && sudo mv minikube /usr/local/bin/
 
-sudo minikube start --vm-driver=none --kubernetes-version=v1.10.0 --extra-config=apiserver.Authorization.Mode=RBAC
+sudo minikube start --vm-driver=none --kubernetes-version=${KUBE_VERSION} --extra-config=apiserver.v=10 --extra-config=kubelet.max-pods=100
 
 sudo minikube update-context
 
-sudo kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
+sudo chown -R ${USER}:${USER} ${HOME}/.kube
```

Comment:
Where is the ${HOME} defined?

Reply:
${HOME} is system env, means the home dir of current user.

Reply:
OK